### ISSUE TITLE
Fragile Artifacts 

### ISSUE THREAD

**Issue by edeksumo** [vcmi/vcmi#4932]
**Describe your proposal**
Fragile Artifacts after turn or battle would disappear from inventory. Maybe with chance to survive.
It would open new possiblity for artifact creation,  powerful one time use artifact. Potions etc. 

**Additional context**

**Comment by krs0** [comment]
So basically Artifacts with Charges? or Charge-Based Artifacts?

I've encountered this proposal for scrolls.

The request to also have a chance not to consume a charge would be something extra?

**Comment by edeksumo** [comment]
Yep, something like that. With ability to specify if the charge is used after battle or after player turn.

**Comment by paddotk** [comment]
I think the inner workings of HoMM3 should not be changed. This might be nice for a mod, but not VCMI itself.

**Comment by IvanSavenko** [comment]
@paddotk 

Correct. No idea whether such "fragile artifacts" would ever be implemented, but it would be added only for use in mods.

VCMI without mods uses and will keep using Heroes 3: SoD mechanics

**Comment by IvanSavenko** [comment]
Implemented in #5720
